hasura.planx.uk/migrations/1720511391883_run_sql_migration/up.sql

```diff
@@ -0,0 +1,32 @@
+CREATE OR REPLACE VIEW "public"."teams_summary" AS SELECT 
+    t.id,
+    t.name,
+    t.slug,
+    t.reference_code,
+    t.settings->>'homepage' as homepage,
+    t.domain as subdomain,
+    ti.has_planning_data as planning_data_enabled,
+    '@todo' as article_4s_enabled,  
+    jsonb_build_object(
+        'helpEmail', ts.help_email,
+        'helpPhone', ts.help_phone,
+        'emailReplyToId', ts.email_reply_to_id,
+        'helpOpeningHours', ts.help_opening_hours
+        ) as govnotify_personalisation,
```

Comment:
This is the new section of this view - constructing the same shape JSONB column from the new `team_settings` column.